### PR TITLE
Feature/wdp191000 19 karta mebla ocena

### PR DESCRIPTION
**Pull request by JagodaBodnar**
Feature:
Currently all products have two full stars. 
image
I changed according to client request:
-on hover and on click there will be no black stars
image
-hovering over 1 star will add color only to that particular one, hovering on 4-th will add color to all four and last one will remain blank, moreover after click picked stars should be freezed.
image
-one product  by default should have all stars coloured
image

I am open to any suggestions cause I am not 100% sure if I understood all correctly.

### CODE REVIEW

File: src/scripts/app.js [hunk 1]
```diff
@@ -1 +1,33 @@
...
+    let action2 = 'remove';
+    for (let element of this.children) {
+      element.classList[action]('hover');
+      element.classList[action2]('full');
+      if (element === e.target) action = 'remove';
+    }
+  });
+}
+
+for (let j = 0; j < allStars.length; j++) {
```
Comment (jacekgalazka) [inline src/scripts/app.js]: może by tu użyć pętli for of ?

File: src/scripts/app.js [hunk 2]
```diff
@@ -1 +1,33 @@
...
+  allStars[j].addEventListener('mouseout', function (e) {
+    let action = 'remove';
+    for (let element of this.children) {
+      element.classList[action]('hover');
+      if (element === e.target) action = 'remove';
+    }
+  });
+}
+
+for (let k = 0; k < allStars.length; k++) {
```
Comment (jacekgalazka) [inline src/scripts/app.js]: j.w.

File: src/scripts/app.js (hunk 1 above)
Comment (JagodaBodnar) [reply src/scripts/app.js]: Okay pewnie.

File: src/scripts/app.js (hunk 2 above)
Comment (JagodaBodnar) [reply src/scripts/app.js]: Okay done.

File: src/scripts/app.js [hunk 3]
```diff
@@ -1 +1,33 @@
...
+const allStars = document.querySelectorAll('.stars');
+
+for (let i = 0; i < allStars.length; i++) {
+  allStars[i].addEventListener('mouseover', function (e) {
+    let action = 'add';
+    let action2 = 'remove';
+    for (let element of this.children) {
+      element.classList[action]('hover');
+      element.classList[action2]('full');
+      if (element === e.target) action = 'remove';
```
[the remark this answers is not in the document]
Comment (JagodaBodnar) [reply src/scripts/app.js]: Jak chcesz mogę to zmienić tak:
 (element === e.target) ? action = 'remove' : '';

[the remark this answers is not in the document]
Comment (Neonides) [reply src/scripts/app.js]: Po prostu
 ```
if (element === e.target) {
 action = 'remove';
}
```

**Pull request merged**